models/sarahb_fizzbuzz.rb

```diff
@@ -0,0 +1,27 @@
+
+class Fizzbuzz
+
+   def initialize number
+      @fizzbuzz = number
+   end
+
+   def string
+     x = @fizzbuzz
+     if x % 3 == 0 && x % 5 == 0
```

Comment:
change to a case statement rather than if/elsif. 
Perhaps move the % == 0 tests outside the case statement - set as a variable and then check if the variable is true/false inside the case statement. Means you'll only be performing each % check once rather than potentially twice